transcode/transcode.go

```diff
@@ -217,3 +223,27 @@ func isInputVideoBiggerThanDefaults(iv clients.InputVideo) bool {
 	}
 	return false
 }
+
+func statsFromProfiles(profiles []clients.EncodedProfile) []RenditionStats {
+	stats := []RenditionStats{}
+	for index, profile := range profiles {
+		stats = append(stats, RenditionStats{
+			Index:  int64(index),
+			Name:   profile.Name,
+			Width:  profile.Width,  // TODO: extract this from actual media retrieved from B
+			Height: profile.Height, // TODO: extract this from actual media retrieved from B
+			FPS:    profile.FPS,    // TODO: extract this from actual media retrieved from B
+		})
+	}
+	return stats
+}
+
+type RenditionStats struct {
+	Index      int64
```

Comment:
I don't think Index should live in here, since it's a property of the containing structure rather than the object itself